deployment/install.sh

```
@@ -45,108 +52,72 @@ check_command() {
+        read -r -p "LLM model identifier (provider/model): " LLM_MODEL </dev/tty
         if [ -z "$LLM_MODEL" ]; then
             print_colored "red" "LLM_MODEL cannot be empty."
         elif [[ ! "$LLM_MODEL" == *"/"* ]]; then
-            print_colored "red" "Invalid LLM_MODEL format. Please use provider/model_name."
+            print_colored "red" "Invalid format. Expected: provider/model (e.g., openai/gpt-4o)"
         else
             break
         fi
     done
 
     local LLM_PROVIDER_RAW=$(echo "$LLM_MODEL" | cut -d'/' -f1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)